io-sim/src/Control/Monad/IOSim/Internal.hs

```diff
@@ -146,6 +152,11 @@ data StmA s a where
   Retry        :: StmA s b
   OrElse       :: StmA s a -> StmA s a -> (a -> StmA s b) -> StmA s b
 
+  SayStm       :: String -> StmA s b -> StmA s b
+  OutputStm    :: Dynamic -> StmA s b -> StmA s b
+
+  LiftStStm    :: StrictST.ST s a -> (a -> StmA s b) -> StmA s b
```

Comment:
What do you need this one for?

Reply:
I removed `liftStStm` in 30 Dec force push.